dts/bindings/spi/spi-device.yaml

```diff
@@ -40,3 +40,21 @@ properties:
     enum:
       - 0
       - 32768
+  spi-cpol:
+    type: boolean
+    description: |
+      SPI clock polarity which indicates the clock idle state.
+      If it is used, the clock idle state is logic high; otherwise, low.
+  spi-cpha:
+    type: boolean
+    description: |
+      SPI clock phase that indicates on which edge data is sampled.
+      If it is used, data is sampled on the second edge; otherwise, on the first edge.
+  spi-hold-cs:
+    type: boolean
+    description: |
+      In some cases, such as BlueNRG-MS, it is necessary for the master to manage
```

Comment:
please avoid specific information in generic bindings docs, ie, what is "BlueNRG-MS"? Just explain what `spi-hold-cs` implies.

Reply:
I wanted to provide a real example; however, if you prefer not to have it, I can remove it.

Reply:
I understand, but for readers without knowledge of what BlueNRG-MS is, the text doesn't add much value. I'd document in the BlueNRG-MS bindings about the need for this option, though.

Reply:
Ok. I removed it.